CHANGELOG.md

```diff
@@ -27,7 +27,7 @@ creation and modification of a status user
 - Fixed bad command usage that was serving as a help for the user for
 the /modify_notify_group command in the invite to notify group flow
 - Improve user prompt when there are no status users to disply the status of
-
```

Comment:
Simple changelog change, however we can probably add more related to more user feedback on invitation to notify group